e2e/support/config.js

```diff
@@ -14,7 +14,7 @@ const isEnterprise = process.env["MB_EDITION"] === "ee";
 const hasSnowplowMicro = process.env["MB_SNOWPLOW_AVAILABLE"];
 const snowplowMicroUrl = process.env["MB_SNOWPLOW_URL"];
 
-const isQaDatabase = process.env["QA_DB_ENABLED"];
+const isQaDatabase = process.env["QA_DB_ENABLED"] === "true";
```

Comment:
Please don't do this.
This will break workflows for a lot of people who set their local ENVs to the boolean `true`.

And even without this fix, this expression will work for strings because of implicit coercion. Any string is truthy.

Reply:
But env can only be a string though? I could revert this fix, but I think using `${{ contains(fromJSON('["postgres", "mongo"]'), inputs.qa_db) }}` is already clean. Otherwise, the condition would also be that if it's `true` return `true` otherwise return empty string which could make the expression more ugly. What do you think?

Reply:
Ok. My bad.
You are right - this solution is even more explicit.

Ignore what I said.

Reply:
I checked that we never pass `QA_DB_ENABLED` anything except `"true"`, so this should be safe. This will only break if people set this locally to something else like `TRUE` though.